tools/generator/src/Generator/SetTargetConfigurations.swift

```diff
@@ -625,6 +629,11 @@ extension Target {
     func linkParamsFilePath() throws -> FilePath {
         return try .internal(internalTargetFilesPath() + "\(name).link.params")
     }
+    
+    // Used to work around CODE_SIGNING_ENABLED = YES in Xcode 14
+    var shouldDisableCodeSigning: Bool {
+        return platform.os == .iOS && product.type == .bundle
+    }
```

Comment:
I used the inverse of your suggestion, since I don't think we can decide in a simple context if it should be _allowed_, but we know exactly when the default should be reversed.